### ISSUE TITLE
Problems with strip_verbatim_indent and Pod::Simple::HTML

### ISSUE THREAD

**Issue by nwellnhof** [perl-pod/pod-simple#85]
Setting `strip_verbatim_indent` on a `Pod::Simple::HTML` object has no effect. This is because of a check for the `VerbatimFormatted` accept code in https://github.com/perl-pod/pod-simple/blob/master/lib/Pod/Simple/BlackBox.pm#L1541:

```
  $line =~ s/^\Q$indent// if $indent
      && !($self->{accept_codes} && $self->{accept_codes}{VerbatimFormatted});
```

`VerbatimFormatted` is added to `Pod::Simple::HTML` in https://github.com/perl-pod/pod-simple/blob/master/lib/Pod/Simple/HTML.pm#L189. So you can work around this issue by unaccepting `VerbatimFormatted`.

**Comment by khwilliamson** [comment]
I'm trying to figure out how to fix this.  I notice that David Wheeler was the one who last changed line 1541 in BlackBox.  I'm hoping he can explain things to me.

**Comment by theory** [comment]
I haven't any idea what `VerbatimFormatted` is for. :-(

**Comment by khwilliamson** [comment]
Thanks for the report.

This should now be fixed in version 3.36 now available from CPAN
